.github/workflows/ci.yml

```diff
@@ -132,7 +132,7 @@ jobs:
       - name: Plot mesh grid
         run: |
           cabal build elastic-buffer-sim:sim
-          cd elastic-buffer-sim && poetry run $(cabal list-bin elastic-buffer-sim:sim) +RTS -M8M -RTS plot complete3 150000 100
+          cd elastic-buffer-sim && poetry run $(cabal list-bin elastic-buffer-sim:sim) +RTS -M128M -RTS plot complete3 150000 100
```

Comment:
I had explicitly put in a tight memory limit here to guard against re-introduction of the memory leak fixed in #181.

Reply:
This indeed was useful for detecting that there still is a memory leak. The 128M are meant to be a temporary workaround for checking that my changes are at least functionally correct. Should we keep the 128M within this PR and resolve the memory leak separately, or do you want to that fixed within this PR as well.

@lmbollen and I already tried to identify the leak on Friday afternoon, but we did not come to a solution yet.

Reply:
Changed it back to `8M`. The space leak got resolved.